### PR TITLE
Fix #226 mybannermaker.com

### LINKED ISSUE #226: mybannermaker.com
@adguard-bot commented on [Fri Jan 24 2020](https://github.com/AdguardTeam/AdguardFilters/issues/48531)

### Issue URL (Incorrect Blocking)
[https://www.mybannermaker.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.mybannermaker.com%2F)
### Comment
Username: @PCOffline
> Most of the content of the website is gone, all fonts are changed and the website is pretty much tore up.

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.3.4
Browser: | Opera
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Web Annoyances Ultralist, <br/>Adblock Warning Removal List<br/><br/><b>Security:</b><br/> Malware Domains Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion, <br/>AdGuard Simplified domain names, <br/>BarbBlock<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard Dutch, <br/>EasyList Hebrew, <br/>YanFung mobile ads
Other extensions used: | ESET Internet Security

### PR DESCRIPTION
**Pull request by Alex-302**
https://github.com/AdguardTeam/AdguardFilters/issues/48531

### CODE REVIEW

File: Filters/exceptions.txt [hunk 1]
```diff
@@ -2,6 +2,8 @@
 ! Once you enable private DNS, Android 9 starts resolving random domains looking like
 ! `*-dnsotls-ds.metric.gstatic.com` (for instance, `a5a6380f-dnsotls-ds.metric.gstatic.com`).
 !
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/226
+||mybannermaker.com^
```
Comment (Sergey-Lyapin) [inline Filters/exceptions.txt]: Forgot @@ at the beginning?

Comment (Alex-302) [reply Filters/exceptions.txt]: no. Wrong file)

**Comment by Alex-302** [comment]
fixed

File: Filters/exceptions.txt (hunk 1 above)
Comment (Alex-302) [reply Filters/exceptions.txt]: moved.

**Pull request merged**